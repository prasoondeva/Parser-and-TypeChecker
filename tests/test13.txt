{
	x : INT;
	x = 1;
	WHILE ( 1 ){
		x = 0;
	}

}