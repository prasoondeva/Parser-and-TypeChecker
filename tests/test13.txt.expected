TYPE MISMATCH 4 C7